react-components/src/architecture/concrete/observations/network.ts

```diff
@@ -5,18 +5,15 @@ import { chunk } from 'lodash';
 import {
   type CreateInstanceItem,
   type DmsUniqueIdentifier,
-  type FdmSDK,
-  type InstanceFilter
+  type FdmSDK
 } from '../../../data-providers/FdmSDK';
 import { type ObservationFdmNode, OBSERVATION_SOURCE, type ObservationProperties } from './models';
 
 import { v4 as uuid } from 'uuid';
 
 export async function fetchObservations(fdmSdk: FdmSDK): Promise<ObservationFdmNode[]> {
-  const observationsFilter: InstanceFilter = {};
-
   const observationResult = await fdmSdk.filterAllInstances<ObservationProperties>(
-    observationsFilter,
+    undefined,
```

Comment:
I was wondering if we can later change the argument list to an object to avoid "forced" undefineds as an argument.
But this is more my opinion on cosmetic changes :D

Reply:
Right :thinking: Good point - I don't have much against it, but I'd rather do it in a subsequent PR